assets/styles/app.scss

```diff
@@ -303,3 +399,44 @@ body {
     }
 }
 
+.load-more-tag {
+    font-family: $textFont;
+    font-size: $h6-font-size;
+    font-weight: 900;
+    cursor: pointer;
+    height: 35.2px;
+
+    &:hover {
+        background-color: $primary !important;
+        color: $secondary !important;
+    }
+
+    &:active,
+    &:focus {
+        scale: 99%;
+        background-color: $light !important;
+        color: $dark !important;
+    }
+
+}
+
+.load-more-tag-mobile {
+    font-family: $textFont;
+    font-size: $h6-font-size;
```

Comment:
Should be done with my custom class.
text-font
Should be done with Bootstrap.
fs-6